farm/metrics.py

```diff
@@ -59,6 +61,40 @@ def compute_metrics(metric, preds, labels):
         raise KeyError(metric)
 
 
+def _correct_bio_encodings(predictions):
```

Comment:
Could you 
- make this function non private (I think people would like to use it regularly) 
- comment on the reasoning why we are doing this and how it is done
Thanks